src/Interface/SteeringDisplay.cpp

```diff
@@ -0,0 +1,328 @@
+#include "SteeringDisplay.h"
+
+#include <stdlib.h>
+#include "Arial12x12.h"
+#include "font_big.h"
+#include "graphics.h"
+#include "AnimationFlashing.h"
+
+/* Display Macros */
+
+// fonts 
+#define SMALL_FONT 	Arial12x12
+#define COOL_FONT 	Neu42x35
+
+// accessories
```

Comment:
All these constants really need to be cleaned up, but not sure that it's within the scope of this PR

Reply:
do you mean all the macros in SteeringDisplay.cpp, or the const char* bitmaps for fonts in all the different header files?

Reply:
@silviu-toderita